projector-plugin/build.gradle.kts

```diff
@@ -44,3 +45,32 @@ intellij {
 (tasks["runIde"] as JavaExec).apply {
   jvmArgs = jvmArgs.orEmpty() + listOf("-Djdk.attach.allowAttachSelf=true", "-Dswing.bufferPerWindow=false")
 }
+
+abstract class GenerateVersionsFile: DefaultTask() {
```

Comment:
If we implement the same logic in different subprojects in the future, we probably need to move this class to `buildSrc` to reuse it. But we can do it later. For now, this class is quite coupled with plugin module in many places.

Reply:
I prefer to do this later.